modules/swagger-codegen/src/main/resources/JavaSpringMVC/apiException.mustache

```diff
@@ -1,10 +1,71 @@
 package {{apiPackage}};
 
+import java.util.Map;
+import java.util.List;
+
 {{>generatedAnnotation}}
-public class ApiException extends Exception{
-	private int code;
-	public ApiException (int code, String msg) {
-		super(msg);
-		this.code = code;
-	}
+public class ApiException extends Exception {
```

Comment:
The calling program still needs to "know" too much about the protocol that's being used. Let's discuss.